models/Hotspot.ts

```diff
@@ -8,6 +8,18 @@ export enum HotspotStatus {
   done = 'finalizat',
 }
 
+export const hotspotStatusList: HotspotStatus[] = [
+  HotspotStatus.toDo,
+  HotspotStatus.inProgress,
+  HotspotStatus.done,
+];
+
+export const HotspotStatusValue = (value: string) => {
+  if (value == HotspotStatus.toDo) return HotspotStatus.toDo;
```

Comment:
we use `===` to make sure the value and the type are equal, `==` only checks the value
https://www.w3schools.com/js/js_comparisons.asp